docker-compose.local.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+
+services:
+  app:
+    image: golang:1.13.1
+    volumes:
+      - .:/opt/kgs
+    working_dir: /opt/kgs
+    user: "${U_ID}:${G_ID}"
```

Comment:
Is this important to have?

Reply:
I think it is a good practice to run docker containers under `user` privileges.
Also, there would be cases, when docker created `go.mod`, `go.sum` with root privileges.

Reply:
@alldroll Since it's inside Docker container, I think it's fine for keeping it as root or we hard code user in the Dockerfile.